### PR TITLE
Refactor loaders and validations

### PR DESCRIPTION
**Pull request by Kevingislason**
## Description

Fixes remaining code review comment for [staging-v6.0.0](https://github.com/Manta-Network/manta-front-end/tree/staging-v6.0.0)

- Validates that transactions are valid to send while in Manta Wallet mode
- Prevents syncs for the inactive wallet mode
- Use production dolphin websockets
- Allow public transfers without Manta Wallet specifically being connected
- Refactor balance logic text
- Memoize context values for privateWalletContext and mantaWalletContext

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [ ] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [x] Updated relevant documentation in the code
- [x] Test code changes manually, and describe your procedure in a comment on this PR
- [x] Mark this PR with the correct milestone

### CODE REVIEW

File: src/config/common.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "VERSION": "5.1.1",
+    "VERSION": "6.0.0",
     "DOWNTIME": false,
     "PAGE_TITLE": "Manta dApp",
```
Comment (jincan39) [inline src/config/common.json:4]: don't know should we improve the PAGE_TITLE, 'Manta Pay' | 'Manta Bridge' | 'Manta Staking'.

File: src/contexts/mantaSignerWalletContext.tsx [hunk 2]
```diff
@@ -26,6 +28,7 @@ const MantaSignerWalletContext = createContext();
 export const MantaSignerWalletContextProvider = (props) => {
   // external contexts
   const config = useConfig();
+  const { usingMantaWallet } = useGlobal();
```
Comment (jincan39) [inline src/contexts/mantaSignerWalletContext.tsx:31]: we might refactor the conditional hooks usage & usingMantaWallet usage in the context in the future. currently i don't have a better idea.

**Review by jincan39 — APPROVED** [review]
Nice Job, appreciated!

**Comment by jasonintju** [comment]
should use useCallback or useMemo for func, then add the exported funcs as dependencies

**Comment by Kevingislason** [comment]
> should use useCallback or useMemo for func, then add the exported funcs as dependencies

Can you comment on the specific functions you think should be memoized?

File: src/utils/display/getZkTransactBalanceText.ts [hunk 3]
```diff
@@ -1,27 +1,67 @@
...
   isPrivate: boolean,
   isInitialSync: boolean,
-  usingMantaWallet: boolean,
   isReady: boolean,
 ) => {
-
   if (apiIsDisconnected) {
     return 'Connecting to network';
-  } else if (usingMantaWallet && isPrivate && !isReady) {
+  } else if (isPrivate && !isReady) {
```
[the remark this answers is not in the document]
Comment (Kevingislason) [reply src/utils/display/getZkTransactBalanceText.ts:12]: Yeah, this doesn't seem to actually be necessary, nice catch

**Review by jasonintju — APPROVED** [review]

**Pull request merged**